api/v1alpha1/releaseplan_types.go

```diff
@@ -58,8 +51,7 @@ type ReleasePlanSpec struct {
 	ReleaseGracePeriodDays int `json:"releaseGracePeriodDays,omitempty"`
 
 	// Target references where to send the release requests
-	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
```

Comment:
Can you not have a kubebuilder annotation if it is optional? It looks like you can, so I think we should keep it

Reply:
FYI I had to change this a little bit to allow setting an empty value or a input that matches the regex. When `Target` is missing in the releasePlan, it can not be created.

The ReleasePlan "tenant-pipeline-rp" is invalid: spec.target: Invalid value: "": spec.target in body should match '^[a-z0-9]([-a-z0-9]*[a-z0-9])?$'
```

Reply:
It is interesting because serviceAccount is also optional and had the exact same kubebuilder validation as target